### DISCUSSION TITLE
翻译后的句子内的词和源语言内词的对应提示, 比如双击选中某个不认识的单词,然后译文里高亮对应的翻译词

### DISCUSSION THREAD

**Discussion by GaoCan702** [immersive-translate/immersive-translate#1325] [Ideas]
### 新功能 | New Feature

1.突出提示原词和译文里词汇的对应

### 为什么您想要添加这些功能 | Why you want to add these features

1.在看中英文对照的时候因为是段落的对照,而不是逐词的对照, 所以对于某个不认识的词汇, 要到译文的寻找, 费眼睛.

**Comment by theowenyoung** [comment]
这个应该不太好实现，因为涉及到第三方翻译服务的支持。
